### DISCUSSION TITLE
Release 5.2.0 Elasticsearch provisioning  issue (Provision/dev/Core/ApplicationElasticSearch)

### DISCUSSION THREAD

**Discussion by gohilamariappan** [Sunbird-Ed/Community#548] [Bugs]
Hi team!
@santhosh-tg 
while provisioning es we got the below error.

######################3
TASK [es6 : set fact plugins_to_install to es_plugins] *************************
fatal: []: FAILED! => {}

MSG:

You need to install "jmespath" prior to running json_query filter
#################

Solution: pip2 install jmespath --> in jenkins server

Regards,
Gohila

**Comment by santhosh-tg** [comment]
@gohilamariappan   **jmespath** should have installed as part of Jenkins setup. Please double check if there any errors during this step.

**Comment by gohilamariappan** [comment]
it seems like no errors in the plugin installtion. have installed this lugin manually in jenkins server. that resolves the issue
